intel_extension_for_transformers/llm/quantization/nn/modules.py

```diff
@@ -94,7 +97,7 @@ def __init__(
         bias=True,
         compute_dtype="fp32",
         compress_statistics=True,
-        weight_dtype="int4_fullrange",
+        weight_dtype="int4_clip",
```

Comment:
why change int4_fullrange to int4_clip

Reply:
int4_fullrange is not a general format, int4_clip is more popular.